diff --git a/node_modules/react-mentions/dist/react-mentions.esm.js b/node_modules/react-mentions/dist/react-mentions.esm.js
index 2efebba..42a1a69 100644
--- a/node_modules/react-mentions/dist/react-mentions.esm.js
+++ b/node_modules/react-mentions/dist/react-mentions.esm.js
@@ -2116,11 +2116,21 @@ var defaultStyle = {
   fontWeight: 'inherit'
 };
 
-var Mention = function Mention(_ref) {
-  var display = _ref.display,
-      style = _ref.style,
-      className = _ref.className,
-      classNames = _ref.classNames;
+var Mention = function Mention({
+  display,
+  style,
+  className,
+  classNames,
+  onAdd = () => null,
+  onRemove = () => null,
+  renderSuggestion = null,
+  trigger = '@',
+  markup = '@[__display__](__id__)',
+  displayTransform = (id, display) => display || id,
+  allowSpaceInQuery,
+  isLoading = false,
+  appendSpaceOnAdd = false
+}) {
   var styles = useStyles(defaultStyle, {
     style: style,
     className: className,
@@ -2154,21 +2164,5 @@ Mention.propTypes = {
   allowSpaceInQuery: PropTypes.bool,
   isLoading: PropTypes.bool
 };
-Mention.defaultProps = {
-  trigger: '@',
-  markup: '@[__display__](__id__)',
-  displayTransform: function displayTransform(id, display) {
-    return display || id;
-  },
-  onAdd: function onAdd() {
-    return null;
-  },
-  onRemove: function onRemove() {
-    return null;
-  },
-  renderSuggestion: null,
-  isLoading: false,
-  appendSpaceOnAdd: false
-};
 
 export { Mention, MentionsInput$1 as MentionsInput };
